website/versioned_docs/version-3.4.1/switch.md

```diff
@@ -23,7 +23,7 @@ import { Switch } from 'react-native-elements';
 ```js
 <Switch value={true} />
 
-<Switch value={false} color="orange">
+<Switch value={false} color="orange"/>
```

Comment:
Make change in, https://github.com/react-native-elements/react-native-elements/blob/next/website/docs/switch.md

Reply:
Linked PR #3041